cli/sawtooth_cli/rest_client.py

```diff
@@ -56,7 +56,11 @@ def send_batches(self, batch_list):
         Returns:
             dict: the json result data, as a dict
         """
-        data_bytes = batch_list.SerializeToString()
+        try:
+            data_bytes = batch_list.SerializeToString()
+        except AttributeError:
+            data_bytes = batch_list
+
```

Comment:
Some of the sawtooth commands send batch lists to the client, and the expect the client to convert them to strings. The intkey message factory converts batch lists to strings itself, so the client needs to be able to handle strings too.

This is a hack really; somebody needs to decide exactly what the interface should be.